.bazelrc

```diff
@@ -45,8 +45,20 @@ build:remote --define=EXECUTOR=remote
 build:remote --remote_executor=grpcs://remotebuildexecution.googleapis.com
 build:remote --remote_timeout=3600
 
+# Minimize what is downloaded
+build:inmemory --experimental_inmemory_jdeps_files
+build:inmemory --experimental_inmemory_dotd_files
+
+# Minimize what is downloaded
+build:toplevel --config=inmemory
+build:toplevel --experimental_remote_download_outputs=toplevel
```

Comment:
Could potentiallt reduce some boilerplate by using ([ref](https://docs.bazel.build/versions/master/command-line-reference.html#flag--experimental_remote_download_toplevel))
```
build:toplevel --experimental_remote_download_toplevel
build:minimal --experimental_remote_download_minimal
```

Reply:
Try it out and fail spectacularly.

Reply:
You are right, looks like those flags are on `master` docs but not in `0.28.1`.